src/back/project/src/api/di-plugin-param.cpp

```diff
@@ -47,7 +47,8 @@ formats::json::Value DiPluginParam::Get(
 {
 	const auto diTypeId = parsePositiveInt(req, "diTypeId");
 	const auto paramId = parsePositiveInt(req, "paramId");
-	res = _s.GetDiPluginParam(diTypeId, paramId);
+	auto spaceId = utils::BoostUuidFromString(req.GetHeader("X-Space-Id"));
```

Comment:
Давай значения доставать из провалидированных параметров, а не напрямую из запроса.
```suggestion
	auto spaceId = utils::BoostUuidFromString(params["X-Space-Id"].As<std::string>());
```

Reply:
ОК (уже не актуально, так как все эти места ушли после рефакторинга на шаблоны).